### PR TITLE
V.0.12.3

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-    proxy_set_header            Host $host;
-    proxy_buffering off;
-    fastcgi_param REMOTE_USER   $remote_user;
-    client_max_body_size        200M;
-    
+
     # Force https
     if ($scheme = http) {
         rewrite ^ https://$server_name$request_uri? permanent;
      }
```
Comment (alexAubin) [inline conf/nginx.conf]:
```suggestion
```

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM but the diff is huge

**Comment by alexAubin** [comment]
Merging to testing

**Pull request merged**